HOWTO_RELEASE.md

```diff
@@ -4,7 +4,7 @@
    - They should cover all (non-GitHub-related) commits (PRs) on the `develop` branch since the most recent release.
    - They should make clear to users whether they might benefit from this release and what backwards incompatibilities they might face.
 
-2. Bump the version numbers in the `develop` branch, in the VERSION file **and** in doc/source/conf.py ([example PR: #1395](https://github.com/GPflow/GPflow/pull/1395)).
```

Comment:
#1395 only changed the `VERSION` file. #1666 also demonstrates updating the `doc` config file.